### PR TITLE
Pass the portforward errors to caller

### PR DESCRIPTION
**Pull request by surajssd**
This is first attempt to add retry mechanism to creation of working client.

### CODE REVIEW

**Comment by surajssd** [comment]
Meant to create a draft PR :man_facepalming:

File: test/components/util/util.go [hunk 1]
```diff
@@ -208,6 +209,7 @@ type PortForwardInfo struct {
 	readyChan     chan struct{}
 	stopChan      chan struct{}
 	portForwarder *portforward.PortForwarder
+	errs          []string
```
Comment (invidian) [inline test/components/util/util.go:212]: Maybe `[]error`?

Comment (surajssd) [reply test/components/util/util.go:212]: The inspiration behind choosing list of string over list of errors is that, I wanted to aggregate the errors and return just one error in the end. Later for aggregating I can just use `strings.Join`, than having to go over the list of errors convert them into string one by one and then joining them using a for loop.

**Comment by surajssd** [comment]
Closing this since disabling autoupdates on the nodes has solved the issue.

**Pull request closed without merging**